app/src/main/res/layout/activity_account.xml

```diff
@@ -248,20 +248,63 @@
                         app:layout_constraintTop_toBottomOf="@id/accountFieldList"
                         tools:visibility="visible" />
 
-                    <ViewStub
```

Comment:
Could not find a good way to use a ViewStub in a ViewBinding, so I inlined the referenced layout. May make AccountActivity a bit slower, but cleaner code is also worth something.

Reply:
I'm not ssure I get it, ViewStub would lazily inflate the ID, how does it work now?

Reply:
ah, you inlined it, gotcha